### PR TITLE
ADD: removed NONET to allow netplay. add libsodium for netplay. add p…

### PR DESCRIPTION
**Pull request by ManuelSch81**
…ossibility to set a server password via diablo.ini file (Phone Book, Password1)

### CODE REVIEW

File: SourceX/DiabloUI/selgame.cpp [hunk 1]
```diff
@@ -218,6 +218,7 @@ void selgame_Diff_Esc()
 void selgame_Password_Init(int value)
 {
 	memset(&selgame_Password, 0, sizeof(selgame_Password));
+	getIniValue("Phone Book", "Password1", selgame_Password, 15);
```
Comment (AJenbo) [inline SourceX/DiabloUI/selgame.cpp]: For now this is a bit of a workaround for not having a onscreen keyboard, so I will consider this a draft PR (not ready to merge, but keeping track of progress)

File: Packaging/OpenDingux/build.sh [hunk 2]
```diff
@@ -60,7 +60,12 @@ make_buildroot() {
 	cp buildroot_${TARGET}_defconfig "$BUILDROOT/configs/${TARGET}_devilutionx_defconfig"
 	cd "$BUILDROOT"
 	make ${TARGET}_devilutionx_defconfig
-	BR2_JLEVEL=0 make toolchain libzip sdl sdl_mixer sdl_ttf
+	if [[ "$TARGET" != "rg350" ]]; then
```
Comment (glebm) [inline Packaging/OpenDingux/build.sh]: This shouldn't be done until performance issues are resolved.
This is tracked in <https://github.com/diasurgical/devilutionX/pull/542> so you can remove these changes from here

File: SourceX/DiabloUI/selgame.cpp (hunk 1 above)
Comment (ManuelSch81) [reply SourceX/DiabloUI/selgame.cpp]: Yes, that's fine as a draft first.

File: Packaging/OpenDingux/build.sh (hunk 2 above)
Comment (ManuelSch81) [reply Packaging/OpenDingux/build.sh]: Couldn't mention any performance issues on my RG350 so far, but i'll analyse it over time in different scenarios.

**Comment by AJenbo** [comment]
Afaik this is now replaced by https://github.com/diasurgical/devilutionX/pull/542

**Pull request closed without merging**